github/repos_autolinks.go

```diff
@@ -0,0 +1,103 @@
+// Copyright 2021 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+)
+
+// AutolinkOptions specifies parameters for RepositoriesService.AddAutolink method.
+type AutolinkOptions struct {
+	KeyPrefix   *string `json:"key_prefix,omitempty"`
+	URLTemplate *string `json:"url_template,omitempty"`
+}
+
+// Autolink represents an autolink reference of a repository and is used by
+// RepositoriesService.ListAutolinks, RepositoriesService.AddAutolink and RepositoriesService.GetAutolink methods.
```

Comment:
If you don't mind, I think I prefer the following instead of these two lines because then, if `Autolink` is used anywhere else, this comment won't become stale and won't need updating:

```
// Autolink represents autolinks to external resources like JIRA issues and Zendesk tickets.
```

Reply:
Sure, done.